dom0/sd-proxy-template-files.sls

```diff
@@ -11,10 +11,4 @@ install-securedrop-proxy-package:
     - pkgs:
       - securedrop-proxy
     - require:
-      - sls: fpf-apt-repo
-
-# Remove the legacy config file location
-remove-legacy-sd-proxy-config:
-  file.absent:
-    - names:
-      - /etc/sd-proxy.yaml
+      - sls: fpf-apt-repo
```

Comment:
noting to myself that this is the last bit of code i need to look into but submitting my review so you can have an opportunity to address comments tomorrow

Reply:
back at it! oh yeah, this is totally fine